diff --git a/node_modules/@devoxa/paddle-sdk/dist/src/helpers/fetch.js b/node_modules/@devoxa/paddle-sdk/dist/src/helpers/fetch.js
index d7aa967..3ab6c7b 100644
--- a/node_modules/@devoxa/paddle-sdk/dist/src/helpers/fetch.js
+++ b/node_modules/@devoxa/paddle-sdk/dist/src/helpers/fetch.js
@@ -1,26 +1,3 @@
-"use strict";
-var __importDefault = (this && this.__importDefault) || function (mod) {
-    return (mod && mod.__esModule) ? mod : { "default": mod };
-};
-Object.defineProperty(exports, "__esModule", { value: true });
-exports.fetch = void 0;
-const form_data_1 = __importDefault(require("form-data"));
-const node_fetch_1 = __importDefault(require("node-fetch"));
-async function fetch(url, options) {
-    const response = await (0, node_fetch_1.default)(url, {
-        method: options.method,
-        body: objectToFormData(options.body),
-    });
-    return response.json();
+export async function fetch(url, options) {
+  throw new Error("Not supported");
 }
-exports.fetch = fetch;
-function objectToFormData(object) {
-    const formData = new form_data_1.default();
-    Object.entries(object).forEach(([key, value]) => {
-        if (typeof value === 'undefined')
-            return;
-        formData.append(key, value.toString());
-    });
-    return formData;
-}
-//# sourceMappingURL=fetch.js.map
\ No newline at end of file
