diff --git a/node_modules/bson/lib/bson.mjs b/node_modules/bson/lib/bson.mjs
index ce98d06..c827ef8 100644
--- a/node_modules/bson/lib/bson.mjs
+++ b/node_modules/bson/lib/bson.mjs
@@ -206,7 +206,7 @@ function nodejsMathRandomBytes(byteLength) {
 }
 const nodejsRandomBytes = await (async () => {
     try {
-        return (await import('crypto')).randomBytes;
+        return require('node:crypto').randomBytes;
     }
     catch {
         return nodejsMathRandomBytes;
