diff --git a/node_modules/bson/lib/bson.mjs b/node_modules/bson/lib/bson.mjs
index c07d7ad..e16db98 100644
--- a/node_modules/bson/lib/bson.mjs
+++ b/node_modules/bson/lib/bson.mjs
@@ -128,9 +128,9 @@ class BSONRuntimeError extends BSONError {
 function nodejsMathRandomBytes(byteLength) {
     return nodeJsByteUtils.fromNumberArray(Array.from({ length: byteLength }, () => Math.floor(Math.random() * 256)));
 }
-const nodejsRandomBytes = await (async () => {
+const nodejsRandomBytes = (() => {
     try {
-        return (await import('crypto')).randomBytes;
+        return require('crypto').randomBytes;
     }
     catch {
         return nodejsMathRandomBytes;
